astropy/utils/xml/writer.py

```diff
@@ -188,13 +182,15 @@ def xml_cleaning_method(self, method='escape_xml', **clean_kwargs):
         current_xml_escape_cdata = self.xml_escape_cdata
 
         if method == 'bleach_clean':
-            if HAS_BLEACH:
-                if clean_kwargs is None:
-                    clean_kwargs = {}
-                self.xml_escape_cdata = lambda x: bleach.clean(x, **clean_kwargs)
-            else:
+            try:
+                import bleach
```

Comment:
Please add a comment to explain why the import is here - it's really helpful to have this kind of comment to avoid regressing in future if someone decides to reorganize the imports.

Reply:
Ok, done.